_layouts/default.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang=fr>
```

Comment:
I would prefer to have quotes even though HTML5 does not make them mandatory, because:

- Syntax consistency is very important for me (less cognitive load when reading and writing code).
- Minimising churn is important for me (less cognitive load when reading diffs).
- As soon as an attribute accepts multiple (thus space-separated) values, you have to quote them, so it is extremely likely that at least one attribute will require quotes.

Is that okay with you? 🙂

Reply:
I'll try to adapt 😉

Reply:
I'm curious how you usually manage that, and willing to try something else if you prefer. Would you have links to example code you have written this way? 🙂